pokemongo_bot/event_handlers/telegram_handler.py

```diff
@@ -56,7 +56,7 @@ def run(self):
                     self.update_id = update.update_id+1
                     if update.message:
                         self.bot.logger.info("message from {} ({}): {}".format(update.message.from_user.username, update.message.from_user.id, update.message.text))
-                        if self.master and self.master not in [update.message.from_user.id, "@{}".format(update.message.from_user.username)]:
```

Comment:
Original code has two way to config the master:
username or chat_id, you changed to only chat_id.